### PR TITLE
Update page-titles-and-section-titles.md

### PR DESCRIPTION
**Pull request by megzehn**
Adding content to "Page titles and section titles" to address these accessibility concerns:

- On pages that have multiple options, add "Option X" before the content.

More background is in this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/52595

### CODE REVIEW

File: src/_content-style-guide/page-titles-and-section-titles.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ Try to keep page titles to 52 characters maximum, with spaces. Use the primary S
 Section and subsection titles (also sometimes called headers and subheads) help organize the page into scannable, user-friendly chunks. They should provide clear guideposts and bring people deeper into the content. 
 
 - Structure section titles with H2s and subsections with H3s, and so on. This provides a natural hierarchy for your content, and helps SEO.
-
-- Try to keep section and subsection titles to 70 characters maximum, with spaces. 
+- In sections that list several ways someone can contact VA or apply for a benefit, use "Option [X]:" in the header. For example, under an H2 that says "How can I apply?" we might have these H3s: "Option 1: Online" and "Option 2: By mail"
```
Comment (laurwill) [inline src/_content-style-guide/page-titles-and-section-titles.md:40]: maybe do "How to apply" for H2 example to align with statement headers style

**Review by laurwill — COMMENTED** [review]
This looks good to me, pending convo about verbs after "Option X:"

**Comment by megzehn** [comment]
Thanks, @laurwill!

@bethpottsVADEPO, could you review this when you get a chance and merge if everything looks OK?

**Review by bethpottsVADEPO — APPROVED** [review]
Looks good, thanks Megan!

**Comment by megzehn** [comment]
Hi @humancompanion-usds, we made a content change to the style guide. Could you review and approve when you have a minute? @bethpottsVADEPO and Danielle have already approved.

Here's the summary:

> In sections where we're listing multiple options for people to get to the same outcome, add "Option X" to the header. For example, if we're describing ways someone can apply for a benefit, we might have this structure:
> 
> [H2] How to apply for [benefit]
> 
> [H3] Option 1: Online
> [H3] Option 2: By mail
> [H3] Option 3: In person

**Review by humancompanion-usds — APPROVED** [review]

**Pull request merged**